src/celeritas/ext/GeantOpticalPhysicsOptions.hh

```diff
@@ -0,0 +1,135 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022-2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ext/GeantOpticalPhysicsOptions.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Types.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+//! WLS time model selection
+enum class WLSTimeProfileSelection
+{
+    none,
+    delta,  //!< Delta function
+    exponential,  //!< Exponential decay
+    size_
+};
+
+//---------------------------------------------------------------------------//
+/*!
+ * Construction options for Geant optical physics.
+ *
+ * These options attempt to default to our closest match to \c
+ * G4OpticalPhysics from Geant4 10.5 onwards.
+ */
+struct GeantOpticalPhysicsOptions
+{
+    //!@{
+    //! \name Optical photon physics
```

Comment:
Is this a placeholder or was it duplicated by mistake?

Reply:
Yes, a duplication be mistake, so removed!